examples/stamping/deployment.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: apps/v1beta1
 kind: Deployment
 metadata:
   name: image-stamping
+  labels:
```

Comment:
is this being used anywhere?

Reply:
Oops nope, I was using it before I split up the targets. Will remove.